### PR TITLE
*: Provide simple example to be run by CI

### PR DESCRIPTION
**Pull request by mxinden**
This commits adds a simple example (`examples/example.rs`) as well as
a Testground manifest (`./manifest.toml`) to run a single Testground
instance, using sdk-rust, connecting to the sync service and signaling
success.

Can be run via:

```
testground plan import --from sdk-rust

testground run single \
         --plan=sdk-rust \
         --testcase=example \
         --builder=docker:generic \
         --runner=local:docker \
         --instances=1 \
         --wait
```

### CODE REVIEW

**Comment by mxinden** [comment]
@galargh as discussed out of band, could you take care of GitHub Actions side? I don't have an opinion on the process, I guess it is easiest if you just push to this branch?

Also any suggestions / changes / ...  to the logic provided thus far here would be very much appreciated.

**Comment by galargh** [comment]
I added a step to CI which runs the example you provided. Here's an example run: https://github.com/testground/sdk-rust/runs/5659084888?check_suite_focus=true

TLDR it works! Installing testground takes quite a long time though. I think it might be time to start distributing testground binaries. I created an issue for this in testground repo - https://github.com/testground/testground/issues/1300

**Comment by mxinden** [comment]
Perfect. Thanks @galargh. This is exactly what we need here.

> Installing testground takes quite a long time though

`4m 10s` should be fine for us for now.

---

(How did you succeed on the first try / commit? :exploding_head:  Asking for a friend.)

**Comment by galargh** [comment]
> (How did you succeed on the first try / commit? 🤯 Asking for a friend.)

1. I did the work on a different branch and only pushed to yours once it was working. Surprisingly though, I only did one tiny mistake the first time around - I tried the last run without `|` in: https://github.com/testground/sdk-rust/blob/5c3bc8b81ccf431d6f847d9d8a25967523298071/.github/workflows/ci.yml#L107
1. That might be because I also use https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml which checks for me if my GitHub Actions ymls are correct.
